### ISSUE TITLE
Implement a Rate Limit

### ISSUE THREAD

**Issue by latentdev** [latentdev/JP-Project#11]
implement rate limiting so twitter doesn't shut us out when we make a certain number requests within a window of time.

**Comment by ghost** [comment]
Should we be selective when pulling Tweets?

**Comment by latentdev** [comment]
no, but I think the way I want to go about this is we keep track of the number of requests we make and then when we reach the limit we switch our credentials out for another set.
